python/nano/test/openvino/pytorch/test_openvino.py

```diff
@@ -303,3 +303,33 @@ def test_openvino_trace_kwargs(self):
                                             )
         with InferenceOptimizer.get_context(ov_model):
             result = ov_model(x)
+
+    def test_openvino_trace_stable_diffusion_unet(self):
+        from diffusers.models import UNet2DConditionModel
+        unet = UNet2DConditionModel(sample_size=64,
+                                    cross_attention_dim=10,
+                                    attention_head_dim=1,
+                                    down_block_types=("CrossAttnDownBlock2D", "DownBlock2D"),
+                                    block_out_channels=(32, 64),
+                                    up_block_types=("UpBlock2D", "CrossAttnUpBlock2D"),
+                                    layers_per_block=1)
+        latent_shape = (2, 4, 8, 8)
+        image_latents = torch.randn(latent_shape, device = "cpu", dtype=torch.float32)
+        encoder_hidden_states = torch.randn((2, 12, 10), device = "cpu", dtype=torch.float32)
+        input_sample = (image_latents, torch.Tensor([980]).long(), encoder_hidden_states, False)
+        unet(image_latents, torch.Tensor([980]).long(), encoder_hidden_states)
+        dynamic_axes= {"sample": [0],
+                        "encoder_hidden_states": [0],
+                        "unet_output": [0]}
+        nano_unet = InferenceOptimizer.trace(unet, accelerator="openvino",
+                                                input_sample=input_sample,
+                                                input_names=["sample", "timestep",
+                                                            "encoder_hidden_states", "return_dict"],
+                                                output_names=["unet_output"],
+                                                dynamic_axes=dynamic_axes,
```

Comment:
I wonder what will happen if we leave this `dynamic_axes` to be autogenerated

Reply:
Just validate it, no need for `dynamic_axes`, `input_names` and `output_names` now. Will update the ut. @cyita

Reply:
If leave this `dynamic_axes` to be autogenerated, then model can't deal with dynamic shape which is showed in uts.